docs/platform/porting/system.md

```diff
@@ -2540,3 +2540,353 @@ Example:
+-  Read and apply provided configuration to the system, apply configuration defined limits.
+-  Generate D-Bus signal and abnormality report, if configured.
+
+#### Configuration
+
+The configuration uses a hierarchical layout which consists of a set of JSON files. The primary file is `/usr/lib/stability-monitor/default.conf`. Additional files can be placed in `/etc/stability-monitor.d`, and the files are read in alphabetical order, for example, `Z.conf` overrides `A.conf`. Configuration files are merged together and not replaced.
+
+In the resulting JSON object, the top-level entries have keys corresponding to programs, for example:
+-  `resourced`
+-  `org.tizen.calendar`
+
+Program name must be application ID for applications and executable name for other system processes.
+
+If a program does not have specific configuration, the appropriate class-wide configuration is used. The available classes are:
+-  **global**  
+     Global section for all processes.
+
+-  **foreground_native_app**  
+     Native applications running in foreground.
+
+-  **background_native_app**  
+     Native applications running in background.
+
+-  **foreground_web_app**  
+     Web applications running in foreground.
+
+-  **background_web_app**  
+     Web applications running in background.
+
+-  **foreground_csharp_app**  
+     .NET applications running in foreground.
+
+-  **background_csharp_app**  
+     .NET applications running in background.
+
+-  **foreground_ui_widget**  
+     Widget applications running in foreground.
+
+-  **background_ui_widget**  
+     Widget applications running in background.
+
+-  **native_service**  
+     Service applications.
+
+-  **other_service**  
+     Other system processes.
+
+The `global` is the rule-set used as a fallback when neither the process-specific nor the relevant class-wide configuration exists. In addition to the rules described as follows, this rule-set must specify the `sampling_rate` value which describes how often to read the process data from the kernel.
+
+Each rule-set specifies a set of parameters applicable to the given process or group. The rule-sets are as follows:
+
+-  **monitor**  
+     Defines whether to monitor a process.
+
+     Default value: `1`
+
+-  **print_current**  
+     Defines whether to print current process resource usage in real-time.
+
+     Default value: `0`
+
+-  **apply_to_children**  
+     Defines whether process rule-set should be applied to all of its children.
+
+     Default value: `0`
+
+-  **kill**  
+     Defines whether to kill the process after detecting abnormality.
+
+     Default value: `0`
+
+-  **report**  
+     Defines whether to create bug report after detecting abnormality.
+
+     Default value: `0`
+
+-  **sampling_rate**  
+     Specifies the sampling rate (Hz). This is the process data update frequency. Maximum allowed value is specified by **sampling_rate** from the `global` section.
```

Comment:
Shouldn't **sampling_rate** be in code styling instead? If no, ignore the comment.